S2VX.Game/Story/Note/S2VXNote.cs

```diff
@@ -11,12 +11,14 @@ public abstract class S2VXNote : CompositeDrawable, IComparable<S2VXNote> {
         public double HitTime { get; set; }
         public Vector2 Coordinates { get; set; } = Vector2.Zero;
         public virtual Approach Approach { get; set; }
-        public float OutlineThickness { get; set; }
-        public Color4 OutlineColor { get; set; }
 
         protected RelativeBox BoxOuter { get; } = new RelativeBox();
         protected RelativeBox BoxInner { get; } = new RelativeBox();
 
+        public Color4 InnerColor { get => BoxInner.Colour; set => BoxInner.Colour = value; }
+        public Color4 OutlineColor { get => BoxOuter.Colour; set => BoxOuter.Colour = value; }
```

Comment:
Don't know if this indirection is really necessary. I think I'd rather just make `BoxInner` and `BoxOuter` public.

Reply:
These used to be dedicated Getter/Setter functions but Intellectus recommended that I change it to this format. I prefer to keep them like this because later on if we decide to change this (like in #262 ), we wouldn't have to change and remember the names of the objects within the S2VXNote whenever we want to do something with the note color.